cdap-watchdog/src/main/java/io/cdap/cdap/logging/appender/LogAppender.java

```diff
@@ -139,6 +140,17 @@ private void addErrorClassificationTags(ILoggingEvent event, Map<String, String>
             ((ProgramFailureException) throwable).getErrorReason());
         modifiableMDC.put(Constants.Logging.TAG_ERROR_TYPE,
             ((ProgramFailureException) throwable).getErrorType().name());
+        String errorCodeType = ((ProgramFailureException) throwable).getErrorCodeType();
+        int errorCode = ((ProgramFailureException) throwable).getErrorCode();
+        String supportedDocURL =
+            ((ProgramFailureException) throwable).getSupportedDocumentationUrl();
+        if (!Strings.isNullOrEmpty(errorCodeType)) {
```

Comment:
NIT : is it possible that `errorCodeType` is null ? but `errorCode` is not null ? 

in that case should be include it ?

Reply:
Nope, it is not possible.

Reply:
on a second thought, I can understand error code might not be always `int` so it is better to cater that

Reply:
done.